examples/test-js/test.js

```diff
@@ -3,7 +3,15 @@ import { handle } from "wasi:http/incoming-handler@0.2.0"
 import { calls, Store } from "fermyon:spin-test-virt/key-value";
 import { OutgoingRequest, Fields } from "wasi:http/types@0.2.0"
 
-export function run() {
+export function run(test) {
+  if (test == "cacheHit") {
+    cacheHit();
+  } else {
+    throw new Error(`Test ${test} not found`);
+  }
```

Comment:
It would be great if there was some global context we could call the function on like `window[test]()` would allow us to if we were running in a browser.